Dockerfile

```diff
@@ -0,0 +1,21 @@
+# nb. We use Alpine as a base image and then install Node separately
```

Comment:
This `Dockerfile` matches the one from the client repo, except that it doesn't install `yarn` because this repo is still using `npm`. It also installs additional tools (`zip`) needed for creating browser extension packages.